devices/philips.js

```diff
@@ -1040,6 +1040,14 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp(),
         ota: ota.zigbeeOTA,
     },
+    {                                           
+        zigbeeModel: ['LTC005'],                      
+        model: '3216131P6',                                                         
+        vendor: 'Philips',
+        description: 'Hue ambiance ceiling',                
+        meta: {turnsOffAtBrightness1: true},                    
+        extend: hueExtend.light_onoff_brightness_colortemp(),                       
```

Comment:
Please add the color temperature range as described on https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Reply:
Hi, @Koenkk,
Thank you for the review and the link to the information. I just updated and pushed to my repo after querying through the mqtt.

Would you take a look and let me know if you want me to reopen this or close this one and open a new PR?